apps/image-focal-point/src/components/AppView/AppView.jsx

```diff
@@ -143,7 +143,7 @@ export class AppView extends Component {
       <>
         <div className={styles.background} />
         <div className={styles.body}>
-          <Typography>
+          <React.Fragment>
```

Comment:
I was thinking that `React.Fragment` is an old method that was replaced with just the empty brackets `<></>` so we probably don't need this? I think `Typography` was basically deprecated in Forma for the same reason, that we just use empty fragment tags now.

Reply:
Can change 👍